docs/blog/newsletter-june-2022.rst

```diff
@@ -0,0 +1,130 @@
+
+Instead of focusing on hours, you might think of story points as a more reliable indicator of effort that helps in the division of work. Finally, remember that story points can be assigned based on factors other than just effort, such as complexity and uncertainty.
+
+When used correctly during team planning, story points can give quick approximations, improve project forecasts, and even provide managers with actual statistics during hiring conversations.
+
+------------------
+What we’re reading
+------------------
+
+The #bipoc group’s been discussing the following materials on diversity, inclusion, and equity. Want to join the conversation? Please join us in the `#bipoc Slack channel <https://writethedocs.slack.com/archives/C016STMEWJD>`_!
+
+A short read: Talent Management describes why `many DEI efforts fall flat and how belonging may be the key <https://www.talentmgt.com/articles/2022/05/10/effectively-creating-belonging-why-diversity-efforts-fall-short/>`__.
+
+A medium read: Want to be more supportive of underrepresented groups but aren't sure how? The Muse offers `seven examples of how to become an ally in the workplace <https://www.themuse.com/advice/what-is-an-ally-7-examples>`__.
+
+A longer read: Now a national holiday in the United States, June 19th is Juneteenth. Want to learn more about it? Oprah `explains the history of Juneteenth <https://www.oprahdaily.com/life/a32893726/what-is-juneteenth/>`__.
+
+----------------
+From our sponsor
+----------------
+
+This month’s newsletter is sponsored by KnowledgeOwl:
+
+.. raw:: html
+
+    <hr>
+    <table width="100%" border="0" cellspacing="0" cellpadding="0" style="width:100%; max-width: 600px;">
+      <tbody>
+        <tr>
+          <td width="75%">
+          <p>In the past, the <a href="https://www.knowledgeowl.com/home/blog">KnowledgeOwl blog</a> has covered a wide range of topics from guest bloggers and staff. Now we're tightening our focus to something we care passionately about: Documentarian Quality of Life.&nbsp; </p>
+          <p>So we owls have come with a request: we need your help. What are your biggest pain points? What deep-dive posts would you like to see?</p>
+          <p>We're also seeking 🦉 owl-themed names 🦉 for our new blog categories (and can offer swag in return). </p>
+          <p>See <a href="https://www.knowledgeowl.com/home/our-new-content-strategy">our latest blog post</a> for more details on this shift and how to send us your suggestions. </p>
+          </td>
+          <td width="25%">
+            <a href="https://www.knowledgeowl.com/home/blog">
+              <img style="margin-left: 15px;" alt="SPONSOR" src="/_static/img/sponsors/knowledgeowl-owl.png">
+            </a>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    <hr>
+
+*Interested in sponsoring the newsletter? Take a look at our* `sponsorship prospectus </sponsorship/newsletter/>`__.
+
+------------------
+Featured job posts
```

Comment:
Just a note that the website shuffles features jobs randomly. I think this is the first time we've had more than 10, but we need to refresh it to get the full list. You can also find the full list in our website sidebar on https://www.writethedocs.org/

Reply:
<ul>
        <li><a href="https://jobs.writethedocs.org/job/743/developer-advocate/">Trunk - Developer Advocate</a></li><li><a href="https://jobs.writethedocs.org/job/736/technical-writer-kotlin/">JetBrains - Technical Writer (Kotlin)</a></li><li><a href="https://jobs.writethedocs.org/job/735/technical-writer-intellij/">JetBrains - Technical Writer (IntelliJ)</a></li><li><a href="https://jobs.writethedocs.org/job/733/technical-writer-software-engineering/">Google - Technical Writer, Software Engineering</a></li><li><a href="https://jobs.writethedocs.org/job/732/technical-writer-cloud-technologies-and-tools/">Google - Technical Writer, Cloud Technologies and Tools</a></li><li><a href="https://jobs.writethedocs.org/job/731/technical-writer-serverless-google-cloud/">Google - Technical Writer, Serverless, Google Cloud</a></li><li><a href="https://jobs.writethedocs.org/job/730/technical-writer-cloud-data-analytics-streams-and-lakes/">Google - Technical Writer, Cloud Data Analytics, Streams and Lakes</a></li><li><a href="https://jobs.writethedocs.org/job/729/senior-manager-technical-writing-developer-acceleration-platforms/">Google - Senior Manager, Technical Writing, Developer Acceleration Platforms</a></li><li><a href="https://jobs.writethedocs.org/job/728/technical-writer-data-analytics-bigquery-google-cloud/">Google - Technical Writer, Data Analytics, BigQuery, Google Cloud</a></li><li><a href="https://jobs.writethedocs.org/job/727/technical-content-strategist-developer-relations-content-google-cloud/">Google - Technical Content Strategist, Developer Relations Content, Google Cloud</a></li><li><a href="https://jobs.writethedocs.org/job/726/developer-evangelist/">Twilio - Developer Evangelist</a></li><li><a href="https://jobs.writethedocs.org/job/725/developer-educator-remote-apj/">Twilio - Developer Educator - Remote APJ</a></li><li><a href="https://jobs.writethedocs.org/job/718/technical-writer/">Wix.com - Technical Writer</a></li><li><a href="https://jobs.writethedocs.org/job/649/technical-writer/">Semaphore - Technical Writer</a></li><li><a href="https://jobs.writethedocs.org/job/717/technical-writer-developer-documentation-fully-remote/">Humanitec GmbH - Technical Writer (Developer Documentation) (fully remote)</a></li><li><a href="https://jobs.writethedocs.org/job/714/product-documentation-manager/">ReadMe - Product Documentation Manager</a></li></ul>

Reply:
I've actually updated the website so it shows all featured jobs 👍